src/components/Header/Header.js

```diff
@@ -2,18 +2,35 @@ import React from 'react';
 import HeaderNav from './HeaderNav/HeaderNav';
 import HeaderUserMenu from './HeaderUserMenu/HeaderUserMenu';
 import headerData from '../../headerData.json';
-import { ReactComponent as DopplerLogo } from '../../doppler-logo.svg';
 
-class Header extends React.Component {
-  render() {
-    return (
-      <header>
-        <DopplerLogo />
+const Header = (props) => {
+  return (
+    <header className="header-main">
+      <div className="header-wrapper">
+        <div className="logo">
+          <span className="ms-icon icon-doppler-logo" />
+        </div>
         <HeaderNav nav={headerData.nav} />
-        <HeaderUserMenu user={headerData.user} />
-      </header>
-    );
-  }
-}
+        <nav className="nav-right-main">
+          <ul>
+            <li>
+              <a href="http://google.com" className="active" data-count="1">
```

Comment:
Perhaps you could you change http://google.com for something more like doppler: https://www.fromdoppler.com/

Reply:
or maybe using _REACT_APP_API_URL_

Reply:
Fixed